dev-proxy/ProxyConfiguration.cs

```diff
@@ -39,4 +39,5 @@ public class ProxyConfiguration : IProxyConfiguration
     public int ApiPort { get; set; } = 8897;
     public bool ShowSkipMessages { get; set; } = true;
     public bool ShowTimestamps { get; set; } = true;
+    public long? Timeout { get; set; }

```

Comment:
Let's add a reference to this new option in https://github.com/dotnet/dev-proxy/blob/main/schemas/v0.25.0/rc.schema.json so that users will get it in their intellisense.